jest.setup.js

```diff
@@ -1 +1,5 @@
-jest.setTimeout(30000);
+const mkdirp = require("mkdirp");
+const rimraf = require("rimraf");
```

Comment:
We want to start from a _tabla rasa_ with testing every time, so I think it's important we delete everything when we start.  The creation of `.unmock` is part of the normal flow.  I don't think we need rimraf, though - another way to do it is to make the testing folder random every time (ie `tmp/<uuid>`).  Can that be the subject of a different PR?

Reply:
Definitely for a different PR. A temp folder would indeed be an ideal solution IMO, and we can do that by passing an argument of where to save the mocks (instead of `.unmock/save`)